packages/components/src/ui/calendar.tsx

```
@@ -0,0 +1,62 @@
+'use client';
+
+import { ChevronLeft, ChevronRight } from 'lucide-react';
+import { DayPicker } from 'react-day-picker';
+
+function Calendar({ className, classNames, showOutsideDays = true, ...props }: React.ComponentProps<typeof DayPicker>) {
+  function cn(arg0: string, className: string | undefined): string | undefined {
+    throw new Error('Function not implemented.');
+  }
+
+  function buttonVariants(arg0: { variant: string }): any {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)